app/models/admin_only/member_app_waiting_reason.rb

```diff
@@ -0,0 +1,25 @@
+module AdminOnly
+  
+  class MemberAppWaitingReason < ApplicationRecord
+
+    validates_presence_of :name_sv
```

Comment:
Separate PR is the way to go, especially since we don't want to expand this PR to included an added association for MA's (to `reason(s)`).